database/seeds/DatabaseSeeder.php

```diff
@@ -1,21 +0,0 @@
-<?php
```

Comment:
@jfernancordova why are the Seeder all deleted? Any reason for that in Laravel8?

Reply:
I'm sorry, with laravel 8 the folder seeds is changed to a new folder called seeders and my IDE moved them to non-versioned files, that's why I don't push new changes.